### PR TITLE
Backend isolation

### PR DESCRIPTION
**Pull request by klin10**
Migration to isolate the server backend script from our javascript front end libraries.

### CODE REVIEW

File: app/js/functions.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // DATASHEET CONFIG
 var DEFAULT_DATA = "data/nst_2011.csv";
 var DEFAULT_TOPO = "data/us-states.topojson";
-var USER_DIRECTORY = "uploader/upload/";
+var USER_DIRECTORY = "../../server/uploader/upload/";
```
Comment (CaseyHillers) [inline app/js/functions.js]: This can be switched to /server/uploader/upload/ as that will point to the root of the project instead.

**Review by CaseyHillers — APPROVED** [review]
Looks good, i'll make the merge

**Comment by CaseyHillers** [comment]
Just waiting on Luke's PR to go through so nothing gets messed up there from this.

File: app/js/index.js [hunk 2]
```diff
@@ -23,11 +23,13 @@ var fields;
 var states;
 
 // DATASHEET CONFIG
+var TOPO_DIRECTORY = "data/";
 var DEFAULT_DATA = "data/nst_2011.csv";
 var DEFAULT_TOPO = "data/us-states.topojson";
 var DATA_DIRECTORY = "../examples/";
-var TOPO_DIRECTORY = "data/";
-var USER_DIRECTORY = "uploader/upload/";
+var PHP_DIRECTORY = "../server/php/";
```
Comment (CaseyHillers) [inline app/js/index.js]: forward slash

File: app/js/functions.js [hunk 3]
```diff
@@ -7,7 +7,6 @@ function getCSVFields(callback, CSV) {
         }
     });
 }
-
```
Comment (CaseyHillers) [inline app/js/functions.js]: we should probably keep this line

File: server/uploader/upload/BLANK [hunk 4]
```diff
@@ -0,0 +1 @@
+BLANK FILE
```
Comment (CaseyHillers) [inline server/uploader/upload/BLANK]: what's this for

**Review by CaseyHillers — CHANGES_REQUESTED** [review]
make php files have same file name format

File: server/uploader/upload/BLANK (hunk 4 above)
Comment (klin10) [reply server/uploader/upload/BLANK]: BLANK FILE

**Review by CaseyHillers — APPROVED** [review]
Looks good, just update of what files need to be moved!

**Pull request merged**